idea/gearexe/frontend/src/features/activity/ui/activity.tsx

```diff
@@ -14,6 +19,9 @@ const Activity = () => {
   const [tabIndex, setTabIndex] = useState(0);
   const [isOpen, setIsOpen] = useState(true);
 
+  const myActivity = useAtomValue(myActivityAtom);
```

Comment:
Activity bar is present at the app layout level - would we need `jotai` at all, if there wouldn't be manual set of users activity?